packages/@datapull/atlassian-origin/index.js

```diff
@@ -8,11 +8,11 @@ class AtlassianOrigin {
     this.username = config.username;
     this.password = config.password;
     this.batchSize = Number(config.batchSize || 50);
-    this.service = config.service;
-    this.includeInactiveUsers = config.includeInactiveUsers || false;
-    if (this.service === 'jira' && this.batchSize > 50) {
+    if (this.batchSize > 50) {
```

Comment:
Still, the if statement is out of sync with the exception message right? The message talks about a 1K limit, but we're checking for 50

Reply:
Resolved :+1: